science/digital_rf/Portfile

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           cxx11 1.1
+PortGroup           python 1.0
+
+github.setup        MITHaystack digital_rf 2.6.0
+name                digital_rf
```

Comment:
The `name` line should be removed because it is already set to that by the `github.setup` line above.